packages/gen-ai-hub/src/orchestration/orchestration-client.test.ts

```diff
@@ -4,8 +4,12 @@ import { HttpDestination } from '@sap-cloud-sdk/connectivity';
 import { mockGetAiCoreDestination } from '../test-util/mock-context.js';
 import { mockInference, parseMockResponse } from '../test-util/mock-http.js';
 import { BaseLlmParametersWithDeploymentId } from '../core/index.js';
-import { CompletionPostResponse } from './client/api/index.js';
+import {
+  CompletionPostResponse,
+  FilteringModuleConfig
+} from './client/api/index.js';
 import { GenAiHubCompletionParameters } from './orchestration-types.js';
+import { createAzureFilter } from './orchestration-filter-utility.js';
```

Comment:
wdy think about:

```suggestion
import { azureContentFilter } from './orchestration-filter-utility.js';
```

The fact that we create a more complex object is somewhat irrelevant for the user, they might instead think that something on the server side is "created".. 


I don't have a strong preference, I am just wondering 😄

Reply:
I like it, applied the changes.